x-pack/plugins/ml/public/application/jobs/new_job/pages/new_job/wizard_steps.tsx

```diff
@@ -81,16 +92,35 @@ export const WizardSteps: FC<Props> = ({ currentStep, setCurrentStep }) => {
       )}
       {currentStep === WIZARD_STEPS.PICK_FIELDS && (
         <Fragment>
-          <Title data-test-subj="mlJobWizardStepTitlePickFields">
-            <FormattedMessage
-              id="xpack.ml.newJob.wizard.stepComponentWrapper.pickFieldsTitle"
-              defaultMessage="Pick fields"
+          <MLJobWizardFieldStatsFlyoutContext.Provider
```

Comment:
To make this more reusable, we could have a new component which provides this context and `EuiFlexGroup` and wraps it around whatever children are passed to it.

Reply:
this would also hopefully remove the `fieldName` and `fieldValue` variables from this file, which feel like they shouldn't belong in this wizard steps file.

Reply:
Updated here [`e130206` (#147322)](https://github.com/elastic/kibana/pull/147322/commits/e1302060a89b3575dc5857ad84f23b39986a0531)